tests/controllers/buyerReview.test.ts

```diff
@@ -0,0 +1,154 @@
+import { MongoMemoryServer } from 'mongodb-memory-server';
+import mongoose from 'mongoose';
+
+import BuyerReview from '../../src/models/buyerReview';
+
+import {
+  mockBuyerUser,
+  mockSellerUser,
+  mockSellerService,
+  mockBuyerRequest,
+  mockBuyerReview,
+} from '../helpers/buyerReview';
+
+import {
+  getAllBuyerReviews,
+  getBuyerReviewById,
+  getBuyerReviewsBymockBuyerUser,
+  getBuyerReviewsBymockSellerUser,
+  createBuyerReview,
+  deleteBuyerReview,
+} from '../../src/controllers/buyerReview';
+import { ObjectId } from 'mongodb';
+
+describe('Controller: BuyerReview', () => {
+  let mongoServer: MongoMemoryServer;
+
+  beforeAll(async () => {
+    mongoServer = await MongoMemoryServer.create();
+
+    await mongoose.connect(mongoServer.getUri());
+
+    await mockBuyerUser.save();
+    await mockSellerUser.save();
+    await mockSellerService.save();
+    await mockBuyerRequest.save();
+    await mockBuyerReview.save();
+  });
+
+  afterAll(async () => {
+    if (mongoServer) {
+      await mongoose.connection.dropDatabase();
+      await mongoose.connection.close();
+      await mongoServer.stop();
+    }
+  });
+
+  it('Should be able to fetch all BuyerReviews', async () => {
+    let getAllBuyerReviewsQuery = await getAllBuyerReviews();
+    expect(getAllBuyerReviewsQuery.status).toBe(true);
+    expect(getAllBuyerReviewsQuery.data).toBeInstanceOf(Array);
+    expect(getAllBuyerReviewsQuery.data?.length).toBe(1);
+    expect(new ObjectId(getAllBuyerReviewsQuery.data?.at(0)?._id)).toEqual(
+      new ObjectId(mockBuyerReview._id)
+    );
+  });
+
+  it('Should be able to fetch BuyerReview by ID', async () => {
+    let getBuyerReviewByIdQuery = await getBuyerReviewById(
+      mockBuyerReview._id.toString()
+    );
+    expect(getBuyerReviewByIdQuery.status).toBe(true);
+    expect(getBuyerReviewByIdQuery.data).toBeInstanceOf(BuyerReview);
+    expect(new ObjectId(getBuyerReviewByIdQuery.data?._id)).toEqual(
```

Comment:
There's no need for new ObjectId in any of the tests. You cna directly say `expect(getBuyerReviewByIdQuery.data?._id).toEqual(mockBuyerReview._id)`